src/clr-addons/quick-list/quick-list.component.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'clr-quick-list',
+  templateUrl: './quick-list.component.html',
+  styleUrls: ['./quick-list.component.scss'],
+})
+export class ClrQuickListComponent implements OnInit {
+  @Input() label: string;
+  @Input() defaultValue: any;
+  @Input() listOfValues: Array<any>;
+  @Input() onCard = false;
+  @Input() required = false;
+  @Input() displayField: string;
+  @Input() placeholder = '-Select an option-';
+  @Output() selectedValuesChanged = new EventEmitter();
+  selectedValuesMap = new Map();
+  index = 0;
+
+  constructor() {}
+
+  ngOnInit() {
+    this.selectedValuesMap.set(this.index, this.defaultValue);
```

Comment:
consider the case that you already have values from backend which you want to show here preselected